packages/itwin/property-grid/scripts/run-e2e-with-docker.js

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+const path = require('path');
+const { exec } = require('child_process');
+const packageName = "property-grid"
+const dockerImageName = `${packageName}-e2e-test-image`
+const dockerContainerName = `${packageName}-e2e-test-container`
+
+const execute = command => new Promise((resolve, reject) => {
```

Comment:
Our coding style suggests wrapping lambda args with parentheses.

```suggestion
const execute = (command) => new Promise((resolve, reject) => {
```

Reply:
Changed